webapp/src/ee/billing/administration/subscriptionPlans/migration/selfhosted/AdministrationSelfHostedEePlanMigrationEdit.tsx

```diff
@@ -0,0 +1,188 @@
+import { Box, Link, TableCell, TableRow, Typography } from '@mui/material';
+import { T, useTranslate } from '@tolgee/react';
+
+import { DashboardPage } from 'tg.component/layout/DashboardPage';
+import { BaseAdministrationView } from 'tg.views/administration/components/BaseAdministrationView';
+import { LINKS, PARAMS } from 'tg.constants/links';
+import { PlanMigrationFormData } from 'tg.ee.module/billing/administration/subscriptionPlans/components/migration/PlanMigrationForm';
+import {
+  useBillingApiMutation,
+  useBillingApiQuery,
+} from 'tg.service/http/useQueryApi';
+import React, { useState } from 'react';
+import { useMessage } from 'tg.hooks/useSuccessMessage';
+import { useHistory, useRouteMatch } from 'react-router-dom';
+import { SpinnerProgress } from 'tg.component/SpinnerProgress';
+import { PaginatedHateoasTable } from 'tg.component/common/table/PaginatedHateoasTable';
+import { PlanMigrationStatus } from 'tg.ee.module/billing/administration/subscriptionPlans/migration/general/PlanMigrationStatus';
+import { EmptyState } from 'tg.component/common/EmptyState';
+import { useDateFormatter } from 'tg.hooks/useLocale';
+import { EditPlanMigrationForm } from 'tg.ee.module/billing/administration/subscriptionPlans/components/migration/EditPlanMigrationForm';
+
+export const AdministrationSelfHostedEePlanMigrationEdit = () => {
```

Comment:
I see that SelfHosted and Cloud edition are very similar, but most of the code is duplicated. You should find a way how to deduplicate it.